src/components/ProfileHeader/index.tsx

```diff
@@ -164,7 +164,7 @@ export const ProfileHeaderComponent: React.FC<Props> = props => {
     confirm({
       title: 'Create Experience Limit Reached!',
       description:
-        'You are currently using lite version of Myriad. You can only create up to 5 experience!Connect Web 3.0 Wallet to create more experience',
+        'You are currently using lite version of Myriad. Connect Web 3.0 Wallet to send friend request.',
```

Comment:
Please move this to locale file instead. Would be better to add the Indonesian version if possible.